src/i18n/languages.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   'en-US': 'English',
   'pt-BR': 'Portuguese (Brazil)',
+  'el-GR': 'Ελληνικά (Greece)',
```

Comment:
Wouldn't be "Greek" more appropriate?

Reply:
it depends ... i have seen both implementations.
for example Windows use " Greek " as lang selection but Android/iOS use  "Ελληνικά".
because it is a messenger app (Program ) it think is more appropriate  .
P.S. if you know  Greek it will not mater :-P

Reply:
That's true. Just thinking of consistency reasons but I guess German (Austrian) or German (Austria) would both be ok as long as we keep one pattern.